scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -1,3 +1,5 @@
+# distutils: language = c++
+
 """
 Routines for performing shortest-path graph searches
 
```

Comment:
There are three outdated references to Fibonacci heaps in the docs (GH won't let me inline the comment but they're at lines 8, 69, 415 currently).